profiles/base/package.use.mask

```diff
@@ -333,7 +333,7 @@ mail-mta/netqmail authcram
 # Matt Turner <mattst88@gentoo.org> (2022-11-16)
 # gnome-music and gnome-photos have not been ported to libsoup:3.0, while
 # other non-slotted dependencies have been.
-gnome-base/gnome-extra-apps tracker
+gnome-base/gnome-extra-apps localsearch
```

Comment:
this is purely educational and no instruction to fix this PR, but: what happens to still existing 45.2 when you do that?